tests/Traits/TPresenterTest.phpt

```diff
@@ -259,6 +259,15 @@ class TPresenterTest extends \Testbench\CustomPresenterTestCase
 		}, 'Tester\AssertException', "field 'test' should be defined as required, but it's not");
 	}
 
+	public function testUserLoggedIn()
+	{
+		$this->logOut();
```

Comment:
User should be logged out by default (remove this line). Look at the scenario I wrote in previous comment:
- test
- login
- test
- logout
- test